src/content-scripts/contentscript.ts

```
@@ -61,4 +60,5 @@ displayStyles(window.location)
     initializeStylesheet();
     chrome.runtime.onMessage.addListener(colorListener);
   })
+  /* eslint-disable-next-line no-console */
```

This file needs a rewrite. Thanks for squashing the errors.